pom.xml

```diff
@@ -23,7 +23,7 @@
         <jackson.version>2.7.4</jackson.version>
         <guava.version>19.0</guava.version>
         <powermock.version>1.7.0RC2</powermock.version>
-        <azuresdk.version>1.2.1</azuresdk.version>
+        <azuresdk.version>1.14.0</azuresdk.version>
```

Comment:
suppose this is already undated in azure-common, right?

Reply:
Have comfirmed with xiangyang, we do not have to have the same sdk version with commons.